msgcntr/messageforums-app/src/webapp/css/msgcntr.css

```diff
@@ -503,11 +497,9 @@ form{margin-bottom:0;}
 	text-decoration: underline;
 	margin-bottom:3px
 }
-.messageCommentBody{
-}
 .threadMovedMsg{
     font-size:1em;
-    color:#555555; 
+    color:#555555;
```

Comment:
@wyu12, here is another colour that could be updated with a variable. @profmikegreene mentioned using "text-color-3" (not the exact name) or the disabled text variable. You'll have to see where this is used and therefore, what the appropriate colour is. Thanks.

Reply:
@profmikegreene @fostersdesign fixed